packages/preview/tabut/1.0.0/typst.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "tabut"
+version = "1.0.0"
+authors = ["Amelia Mowers <https://github.com/Amelia-Mowers>"]
+license = "MIT"
+description = "Display data as tables."
+entrypoint = "lib.typ"
+repository = "https://github.com/Amelia-Mowers/tabut"
+keywords = ["formatting", "table", "tables", "data", "display"]
+exclude = ["README.pdf", "doc/compiled-snippets/*"]
```

Comment:
The example snippets should probably also be excluded. The bash scripts are less problematic because they are small, but would ideally also not become part of the package.

Reply:
Thank you!
Added example snippets to exclude and the the bash scripts.